packages/discord.js/src/structures/PollAnswer.js

```diff
@@ -64,6 +75,15 @@ class PollAnswer extends Base {
     return this.client.emojis.resolve(this._emoji.id) ?? new Emoji(this.client, this._emoji);
   }
 
+  /**
+   * Whether this poll answer is a partial.
+   * @type {boolean}
+   * @readonly
+   */
+  get partial() {
+    return this.poll.partial || (this.text === null && this.emoji === null);
```

Comment:
Isn't the check for emoji === null redundant? Answers can have no emoji, but we know for a fact that its a partial one if we have no cached text

Reply:
Presumably, poll answers might only contain an emoji in the future(?). Just future-proofing I guess
> `text` should always be non-null for both questions and answers, but please do not depend on that in the future.

https://discord.com/developers/docs/resources/poll#poll-media-object-poll-media-object-structure